content/rc/databases/configuration/clustering.md

```diff
@@ -85,11 +85,14 @@ are supported, with the following limitations:
     and pipelining are supported with Redis Cloud cluster
     like if it were a non-cluster DB.
 
-## Changing the hashing policy
+## Managing the hashing policy
```

Comment:
Relrefs that previously pointed to `#changing-the-hashing-policy` will also need to be updated unless you want to create a custom alias to match the old heading.
```suggestion
## Manage the hashing policy
```
OR
```suggestion
## Manage the hashing policy {#changing-the-hashing-policy}
```